zooniverse/front-end-monorepo
# Add support for Turkish translations

## Package
app-project
lib-classifier
lib-react-components

## Linked Issue and/or Talk Post
Follow up to added `tr` support in PFE: https://github.com/zooniverse/Panoptes-Front-End/pull/6160

## Describe your changes
- Added converted dictionaries to all three packages (via [pfe-fem-translation-keys](https://github.com/zooniverse/pfe-fem-translation-keys)).
- Added `tr` to i18n configuration files
- When locale is set to Turkish and text is transformed to uppercase, the Turkish "i" is applied whether the uppercase word is actually Turkish or not. As a result, I had to override the username label in SignedInUserNavigation as well as the username is TeamMember to always have `lang="en"` because a username should never display characters other than the latin alphabet.
- I updated lib-classifier's /dev localeMenu to match app-project's localeMenu.

## How to Review
- Make sure to yarn bootstrap
- Visit a project that uses Survey Task because that particular page will pull keys from all 5 translation dictionaries of the currently selected locale. Examples:
https://local.zooniverse.org:3000/projects/tr/wildcam/wildcam-darien/classify/workflow/3033?env=production
https://local.zooniverse.org:3000/projects/tr/markb-panoptes/survey-task-test-project/classify/workflow/3611?env=staging

# Checklist
_PR Creator - Please cater the checklist to fit the review needed for your code changes._
_PR Reviewer - Use the checklist during your review. Each point should be checkmarked or discussed before PR approval._

## General

## General UX
Example Staging Project: [i-fancy-cats](https://local.zooniverse.org:3000/projects/brooke/i-fancy-cats)
  - Can be used with a screen reader [BBC guide to VoiceOver](https://bbc.github.io/accessibility-news-and-you/assistive-technology/testing-steps/voiceover-mac.html)
  - Can be used from the keyboard [WebAIM guide to keyboard testing](https://webaim.org/techniques/keyboard/#testing)
  - It is passing accessibility checks in the storybook

## New Feature
